src/main.jsx

```diff
@@ -34,6 +37,14 @@ const router = createBrowserRouter([
         path: "signUp",
         element: <SignUp />,
       },
+      {
+        path: "employer-jobpostings",
```

Comment:
FYI, these paths will be modified by someone else when the relevant page has been done